### PR TITLE
Mark 2.0.2 release

### PR DESCRIPTION
**Pull request by dagguh**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,7 +21,10 @@ Adding a requirement of a major version of a dependency is breaking a contract.
...
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/workspace/compare/release-2.0.1...master
+[Unreleased]: https://github.com/atlassian/workspace/compare/release-2.0.2...master
+
+## [2.0.2] - 2019-11-04
+[2.0.2]: https://github.com/atlassian/workspace/compare/release-2.0.1...release-2.0.2
 
 ### Fixed
 - Support working in a Git Worktree folder. Fix [JPERF-667].
```
Comment (mzyromski-atlassian) [inline CHANGELOG.md:30]: As a drive-by-change, could you fix this broken link?

**Review by mzyromski-atlassian — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dagguh) [reply CHANGELOG.md:30]: - [x] fix this broken link

**Review by mzyromski-atlassian — APPROVED** [review]
